CONTRIBUTING.md

```diff
@@ -222,3 +222,29 @@ To run this script:
   4. Run `./scripts/create_issues_versionchange_canonical.sh -t . -s --spec-path path_to_problem_specifications` from the root of this repository and follow the directions.
   
   5. If you submitted new issues, please check these submissions on the [issues page](https://github.com/exercism/java/issues).
+
+## Checking exercises are implemented and submit new issues
+
+There is [a script which allows you to submit new issues](https://github.com/exercism/java/blob/master/scripts/create_issues_new_exercise.sh) to this repo with generic title, description and labels if am unimplemented exercise is detected.
```

Comment:
I think this first sentence doesn't quite make it clear enough what the script is for, it kind of sounds like it's for issues in general, whereas it's meant to be for opening issues for unimplemented exercises I believe? If that's the case then it might be worth rewording it to something like:
"There is a script which allows you to easily check if there are any exercism exercises which haven't been implemented in the Java track, and create issues for those exercises if there are any."
Not necessarily exactly that, but something that makes it clear that this is for unimplemented exercises, if that makes sense? 🙂